src/libmongoc/tests/test-conveniences.h

```diff
@@ -271,4 +271,6 @@ semver_to_string (semver_t *str);
    for (bson_iter_init (&(iter_varname), (bson)); \
         bson_iter_next (&(iter_varname));)
 
+#define test_set_error(error, ...) bson_set_error (error, 1, 1, __VA_ARGS__)
```

Comment:
I'm not sure if the ones are arbitrary or meant to refer to a specific error domain and code. Could we name the magic numbers `1` and `1` with the error domain and code enum values, respectively? Or if there are no corresponding enum values maybe a comment would be helpful?